repo/packages/H/hdfs/5/package.json

```diff
@@ -14,6 +14,6 @@
   ],
   "preInstallNotes": "In order for HDFS to start successfully, it requires a minimum of five nodes; each with at least 2 CPU shares and 8GB of RAM available for use by the HDFS Service.\nNote that the service is experimental and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
   "postInstallNotes": "DC/OS HDFS Service is being installed.\n\n\tDocumentation: https://docs.mesosphere.com/1.7/usage/service-guides/hdfs/\n\tIssues: https://dcosjira.atlassian.net/projects/HDFS/issues",
-  "postUninstallNotes": "DC/OS HDFS Service has been uninstalled.\nPlease follow the instructions at http://docs.mesosphere.com/services/hdfs/#uninstall to remove any persistent state if required.",
+  "postUninstallNotes": "DC/OS HDFS Service has been uninstalled.\nPlease follow the instructions at http://docs.mesosphere.com/1.7/usage/service-guides/hdfs/#uninstall to remove any persistent state if required.",
```

Comment:
I had checked, it looks like there aren't 1.8 versions of the docs yet (/current/ => 404)